lib/navigation.php

```diff
@@ -323,7 +336,7 @@ function gutenberg_output_block_menu_item_custom_fields( $item_id, $item ) {
 		<p class="field-content description description-wide">
 			<label for="edit-menu-item-content-<?php echo $item_id; ?>">
 				<?php _e( 'Content', 'gutenberg' ); ?><br />
-				<textarea readonly><?php echo esc_textarea( trim( $item->content ) ); ?></textarea>
+				<textarea readonly name="menu-item-content[<?php echo $item_id; ?>]"><?php echo esc_textarea( trim( $item->content ) ); ?></textarea>
```

Comment:
Seeing some square brackets in the in the interpolation for the `name` here:
<img width="400" alt="Screenshot 2020-08-18 at 5 39 26 pm" src="https://user-images.githubusercontent.com/677833/90497801-156dd280-e17a-11ea-8cac-3880163c675c.png">

Reply:
That's correct. PHP interprets form values with `[]` as an array.

https://www.php.net/manual/en/reserved.variables.post.php#87650